core/tauri/src/window.rs

```diff
@@ -393,12 +393,47 @@ impl<'a, R: Runtime> WindowBuilder<'a, R> {
   }
 
   /// Whether the window is resizable or not.
+  /// When resizable is set to false, native window's maximize button is automatically disabled.
   #[must_use]
   pub fn resizable(mut self, resizable: bool) -> Self {
     self.window_builder = self.window_builder.resizable(resizable);
     self
   }
 
+  /// Whether the window's native maximize button is enabled or not.
+  /// If resizable is set to false, this setting is ignored.
+  ///
+  /// See [`Window::set_maximizable`] for details.
+  ///
+  /// [`Window::set_maximizable`]: crate::window::Window::set_maximizable
```

Comment:
same, embed the docs here

Reply:
[Fixed](https://github.com/tauri-apps/tauri/pull/6665/commits/480176869123760ba1d9ff4543b6629fb9942762)